news/_posts/2022-01-05-multistore-configuration-forms.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "Multistore configuration forms"
+subtitle: "How multistore configuration forms got better with PrestaShop 1.7.8"
+date: 2022-01-05 
+authors: [matthieu-rolland]
+icon: icon-puzzle
+tags: [multistore, contribute]
+---
+
+Did you know that PrestaShop has a cool feature called "multistore", which is the ability to manage multiple shops from the same back office ?
+
+One of the back-office aspects of this feature is that many configuration forms can have different values for different shops, which is dealt with using shop contexts.
+
+## How it works and what was less than stellar about it
+
+In a configuration form, a merchant can set different values for different shop contexts, of which there are three, in this order of inheritence:
+
+- All shop context
+- Group shop context
+- SIngle shop context
+
+![Shop contexts](/assets/images/2022/01/shopcontexts.png)
+
+When a value is not set for a context, it inherits the value of the parent context. Consequently, the "All Shops" context values can be considered a default values. 
+
+Unfortunately, before Prestashop 1.7.8, this mechanism was not obvious on the forms in question, and it was not easy to see at a glance which form fields were modified or not for the current context.
```

Comment:
It's hard to write this without sounding nasty 😅 

Suggestion:
```suggestion
We listened to merchants feedback and were told this mechanism was not obvious, and it was not easy to see at a glance which form fields were modified or not for the current context.
```
I put the "bouuuuh ! it's not good" part on user feedback 😏  so they're judging not us